rudder-web/src/main/webapp/secure/administration/policyServerManagement.html

```diff
@@ -28,7 +28,21 @@
   select {
     min-width:50px
   }
-  .defaultButton span.ui-button-text {
+  .defaultButton span.ui-button-text {        <div class="col-xs-12 callout-fade callout-warning">
```

Comment:
I don't think you want to add that in a css entry !! or i don't understand something :)